internal/crossdock/server/yarpc/server.go

```diff
@@ -45,13 +48,18 @@ func Start() {
 	if err != nil {
 		log.Panicf("failed to build ChannelTransport: %v", err)
 	}
+	grpcListener, err := net.Listen("tcp", ":8089")
```

Comment:
Will comment more on this later, but this is re https://github.com/yarpc/yarpc-go/pull/866 and https://github.com/yarpc/yarpc-go/pull/897